spec/test/stored_queries/test_query.py

```diff
@@ -102,7 +118,10 @@
 
 
 def use_sort(search_text):
-    """Determine whether to use the sorting or non-sorting query"""
+    """
+    Determine whether to use the sorting or non-sorting stored query for the new query.
+    Smaller search texts' results will not be sorted on.
```

Comment:
There's a handy VSCode extension that I've been using to do this - "autoDocstring: VSCode Python Docstring Generator"